src/solvesystems.jl

```diff
@@ -30,3 +30,37 @@ function solve(system::ODESystem, timespan; solver=AutoVern9(Rodas5()), kwargs..
 
     return sol
 end
+
+"""
+   `variable_connections(systems::Vector{ODESystem})`
+Return the set of variable equalities establishing the connections between the variables in the ODE systems included in the vector `systems`. The function assumes that no two distinct variables have the same name (even if they are defined in different ODE systems).
+"""
+function variable_connections(systems::Vector{ODESystem})
+    @variables t
+    @named _model = ODESystem([], t)
+    @named model = ModelingToolkit.compose(_model, systems)
+    connection_eqs::Set{Equation} = Set{Equation}()
+    var2sys::Dict{String,String} = Dict{String,String}()
+    var2fullvar = Dict()
+    g = variable_dependencies(model)
+    al = g.fadjlist
+    for u in 1:lastindex(al)
```

Comment:
To be consistent with line 56: 
```suggestion
    for u in eachindex(al)
```